content/toolsAndResources/nge/ngeBlocks.md

```diff
@@ -0,0 +1,1387 @@
+
+![Greater Than node](/img/tools/nge/greaterThanNode.jpg)
+
+### GreaterThan
+This logical node tests if the Float value in the left port is greater than the Floating point value in the right port. If the test returns true, the value of ifTrue is passed otherwise the value of ifFalse is passed. 
+
+#### Inputs
+- **left** is connected to a node providing a Float value to be used in the test.
+- **right** can be connected to a node providing a Float value to be used in the test or a constant value can be set in the input field of the properties panel for this node.
+- **ifTrue** can be connected to any data type including geometry but needs to be the same data type as connected to ifFalse and will determine the data type for the output port on this node. If nothing is connected, the value assumed by ifTrue is 1.0.
+- **ifFalse** can be connected to any data type including geometry but needs to be the same data type as connected to ifTrue and will determine the data type for the output port on this node. If nothing is connected, the value assumed by ifFalse is 0.0.
+
+#### Outputs
+- **output** is the value attached to ifTrue if the test returns true or the value attached to ifFalse if the test returns false. If one or both ports for ifTrue and ifFalse are not connected, the output passes 1.0 if the test returns true and 0.0 if the test returns false.
+
+![Less Or Equal node](/img/tools/nge/lessOrEqualNode.jpg)
+
+### LessOrEqual
+This logical node tests if the Float value in the left port is less than or equal to the Floating point value in the right port. If the test returns true, the value of ifTrue is passed otherwise the value of ifFalse is passed. 
+
+#### Inputs
+- **left** is connected to a node providing a Float value to be used in the test.
+- **right** can be connected to a node providing a Float value to be used in the test or a constant value can be set in the input field of the properties panel for this node.
+- **ifTrue** can be connected to any data type including geometry but needs to be the same data type as connected to ifFalse and will determine the data type for the output port on this node. If nothing is connected, the value assumed by ifTrue is 1.0.
+- **ifFalse** can be connected to any data type including geometry but needs to be the same data type as connected to ifTrue and will determine the data type for the output port on this node. If nothing is connected, the value assumed by ifFalse is 0.0.
+
+#### Outputs
+- **output** is the value attached to ifTrue if the test returns true or the value attached to ifFalse if the test returns false. If one or both ports for ifTrue and ifFalse are not connected, the output passes 1.0 if the test returns true and 0.0 if the test returns false.
+
+![Less Than node](/img/tools/nge/lessThanNode.jpg)

```

Comment:
The picture does not show up.

Reply:
fixed in https://github.com/BabylonJS/Documentation/pull/963/commits/81ddf999ffd065a130437ae1d3214eceb956179e